File: 99_bottles.js
function bottleSong(num) {
  while (num > 0) {
    if (num > 1) {
    console.log(`${num} bottles of beer on the wall, ${num} bottles of beer.
    Take one down and pass it around, ${(num = num-1)} bottles of beer on the wall.`)
    } else if (num === 1){
      console.log("1 bottle of beer on the wall, 1 bottle of beer. Take one down and pass it around, 0 bottles of beer on the wall.")
      break;
    }
  } 
}
console.log(bottleSong(2));

File: 99_bottles.py
def bottle_song(num):
	for x in range(num,0,-1): #num is the input we give and we want to create a step range that decrements by 1
		if x == 1:  #change to no more bottles
			print (f"1 bottle of beer on the wall, 1 bottle of beer. Take one down and pass it around, no more bottles of beer on the wall!")
		
		elif x ==2: #change to singular 
			print (f"2 bottles of beer on the wall, 2 bottles of beer. Take one down and pass it around, 1 bottle of beer on the wall.")
			
		elif x > 1: #exercise for loop 
			print (f"{x} bottles of beer on the wall, {x} bottles of beer. Take one down and pass it around, {x-1} bottles of beer on the wall.")


(bottle_song(50))
